### DISCUSSION TITLE
New Sky Culture is Created in Marathi

### DISCUSSION THREAD

**Discussion by santoshasalunke** [Stellarium/stellarium#2987] [Show and tell]
Hi

I am Santosh Salunke from Pune, India.
Created a sky culture in Marathi Language (https://en.wikipedia.org/wiki/Marathi_language).
This sky culture have Marathi names of modern constellations, planets and few stars in Devnagari script. Costellation names were coined by Balshashtri Jambhekar in his magazine "Digdurshun" in year 1842. He translated these names from English to Marathi. Whereas Planet and starnames are traditional.

I want to make this sky culture available to everyone using desktop or app.

Please let me know whom shall I send files to?

Thank you

**Comment by gzotti** [comment]
Thank you for your interest. 

Is this a sky culture with actual figures different from the regular ones? Or just a translation of the "Modern" (formerly "Western") default skyculture? In the latter case, simple translations are handled by Transifex. See our FAQ in the wiki. You may just need to request a new language "Marathi", become supervisor and translate. 

If it is really genuine, create files following the User Guide, probably code as "traditional", write a good description (who, where, what, when, ...), decide for a fitting license, and send a "pull request".

**Comment by santoshasalunke** [comment]
Hi

Thank you for your reply and support.

This is not just a translation, this sky culture includes
1) Constellation names coined in 1842 by an Indian mathematician and
astronomer Balshastri Jambhekar. Some of these are translated from English
names and some are from Indian traditional astronomy and mythology.
2) Planets and star names are from traditional Indian astronomy.
3) All names are in Indian Devanagari script.

I tried creating a "Pull Request" but I'm not sure which "compare" to use
and have no idea what is standard procedure.
I am attaching sky culture files in a Zip file. Is it possible for someone
to create Pull request for me?

Description:
This sky culture is prepared with Marathi names of modern constellations,
planets and stars. Marathi names in Devnagari script are in "Native" list,
whereas IAU names are in "Translated". Constellation names in Marathi were
coined by Balshashtri Jambhekar (1812-46) and published in his magazine
"Digdurshun" in year 1842. He translated these names from English to
Marathi.


Thanks and Regards
Santosh Salunke

**Comment by gzotti** [comment]
(Continued in PR #2989. This is however malformatted. Expecting another PR. )

**Comment by sanketgarade** [comment]
sorry for hijacking this thread but since I couldn't find a way to email the author, I am posting here.

@santoshasalunke

नमस्कार 

मी संकेत, पुण्याहून.
मी पण stellarium वर काम करत आहे.
stellarium मध्ये भारतीय sky culture मधल्या खवस्तूंची यादी वाढवण्याचा व त्यात मराठी भाषा जोडण्याचे काम मी करत आहे.
मदतीसाठी मी "हा तारा कोणता" हे पुस्तक वापरत आहे.

तुम्हाला ह्यात मदत करायची इच्छा असेल तर मला कळवा.
माझा ईमेल पत्ता माझ्या गिटहब प्रोफाइल पानावर तुम्हाला सापडेल.
